packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -1,115 +1,100 @@
 <template>
-  <div class="sf-header" :class="{ 'is-sticky': sticky, 'is-hidden': hidden }">
-    <div class="sf-header__wrapper">
-      <header ref="header" class="sf-header__header">
-        <!--@slot Use this slot to replace logo with text or image-->
-        <slot name="logo" v-bind="{ logo, title }">
-          <SfLink link="/">
-            <SfImage
-              v-if="logo"
-              :src="logo"
-              :alt="title"
-              class="sf-header__logo"
+  <header ref="header" v-click-outside="closeHandler" class="sf-header">
+    <!--@slot Use this slot to replace logo with text or image-->
+    <slot name="logo" v-bind="{ logo, title }">
+      <SfLink link="/">
+        <SfImage v-if="logo" :src="logo" :alt="title" class="sf-header__logo" />
+        <h1 v-else class="sf-header__title">{{ title }}</h1>
```

Comment:
What do you think about using our SfHeader component? :)

Reply:
I add it.